CONTRIBUTING.md

```diff
@@ -29,6 +31,16 @@ Console](https://console.cloud.google.com/apis/library).
 
         npm test
 
+### Running the tests for a Pull Request
+
+When a Pull Request is opened, reopened, or has new commits pushed the tests will be run.
+
+If the tests for a sample change do not run, they can be triggered by adding the `actions:force-run` label.
+
+If tests need to be triggered a second time, manually remove and re-add this label.
+
+The label will not be automatically removed. Please remove it before merging the Pull Request.
```

Comment:
nit: The automatic clean-up of labels is currently disabled. Please remove the `actions:force-run` before merging the Pull Request. 

subtle change, make it explicit that we are intentionally not automatically removing right now. I'm in favor of repeating the exact label to take action on as I'm not assuming folks are not bouncing to exactly the area that they are looking to follow instructions for.

Reply:
```suggestion
The automatic clean-up of labels is currently disabled. Please remove the actions:force-run before merging the Pull Request.
```